backports/py-datadog/APKBUILD

```diff
@@ -0,0 +1,46 @@
+# Contributor: Seiga Kiribayashi <skiribayashi@seqsense.com>
+# Maintainer:
+pkgname="py-datadog"
+_pkgname="datadogpy"
+pkgver="0.35.0"
+pkgrel=0
+pkgdesc="The Datadog Python library"
+url="https://github.com/DataDog/datadogpy"
+arch="noarch"
+license="The 3-Clause BSD"
+options="!check"
+_py2deps="python2"
+_py3deps="python3"
+makedepends="python2-dev python3-dev py-setuptools"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="$pkgname-$pkgver.tar.gz::https://github.com/DataDog/$_pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$_pkgname-$pkgver"
+
+build() {
+	python2 setup.py build
+	python3 setup.py build
+}
+
+package() {
+	depends="py2-${pkgname#py-} py3-${pkgname#py-}"
+	install -D -m 644 LICENSE \
+		"$pkgdir"/usr/share/licenses/$pkgname/LICENSE
+}
+
+_py() {
```

Comment:
https://github.com/seqsense/aports-ros-experimental/blob/dc9690604a4b95e4478142ea2d860df5c0362c03/backports/py-rosinstall/APKBUILD#L45-L47
`install_if` should be added and updating `pkgdesc` is recommended.